draft-ietf-webpush-protocol.xml

```diff
@@ -465,6 +465,9 @@ Location: https://push.example.net/message/qDIYHNcfAIPP_5ITvURr-d6BGt
           from the push service when a push message is delivered and then acknowledged
           by the user agent.
        </t>
+       <t>
+          A push service MUST support the processing of push message receipts.

```

Comment:
What about a smaller change:

**If the push service accepts the message for delivery, then it returns a 202 (Accepted) response.** A URI for the push message resource that was created in response to the request MUST be returned in the Location header field. The push service MUST also provide a URI for the receipt subscription resource in a link relation of type "urn:ietf:params:push:receipt".